scripts/balancing.lua

```diff
@@ -2008,7 +2008,7 @@ for i,spec in ipairs({
                   end
                end
             end
-            if ply ~= GetThisPlayer() then
+            if ply ~= GetThisPlayer() then --this is annoying because you trigger cooldown again, if not troops around
```

Comment:
Yeah, and that's why we *cannot* use cooldown for this purpose

Reply:
agree